src/ecr/process.cr

```diff
@@ -1,2 +1,14 @@
 require "ecr/processor"
-puts ECR.process_file(ARGV[0], ARGV[1])
+
+filename = ARGV[0]
+buffer_name = ARGV[1]
+
+begin
+  puts ECR.process_file(filename, buffer_name)
+rescue ex : Errno
+  raise ex unless {Errno::ENOENT, Errno::EISDIR}.includes?(ex.errno)
```

Comment:
why?
In my coding-style I tend to discard "invalid" values (early returns), then have the content of what  want to do in the method/rescue/anything.